controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -639,15 +649,15 @@ func (r *StorageConsumerReconciler) reconcileCephClientRBDNode(
 		if err := controllerutil.SetOwnerReference(additionalOwner, cephClient, r.Scheme); err != nil {
 			return err
 		}
+		util.AddAnnotation(cephClient, util.GenerationAnnotationKey, currGeneration)
```

Comment:
According to the design, should be a label not an annotation
Please apply everywhere